Parser/ABOParser.php

```diff
@@ -317,7 +336,7 @@ protected function parseTransactionLine($line)
         $transaction->setConstantSymbol($constantSymbol);
 
         # Counter account number
-        $counterAccountNumber = substr($line, 19, 6) . '-' . substr($line, 25, 10);
+        $counterAccountNumber = $this->bankAccountParser->formatBankAccountNumber(substr($line, 19, 6), substr($line, 25, 10));
```

Comment:
Conversion from internal format should be used here too for banks that use internal format

Reply:
Well I've not found an example of bank account. I would add it if we have an example. Otherwise we should leave it as it is?